libs/search-widget/src/components/search-input/SearchInput.svelte

```diff
@@ -232,41 +232,60 @@
   </div>
 
   {#if $filters.length > 0}
-    <div class="filters-container" bind:this={filterContainerElement}>
+    <div
+      class="filters-container"
+      bind:this={filterContainerElement}>
       {#each $filters as filter (filter.key)}
         {#if filter.type === 'creation-start'}
           <Chip
             removable
             color={entitiesDefaultColor}
-            on:remove={() => creationStart.set(undefined)}>
-            {$_('input.from')} {filter.value}
+            on:remove={() => {
```

Comment:
Instead of putting JS code directly in all these `on:remove` events, maybe we should manage everything in the `search()` function, allowing `filter` as optional parameter.

That would be cleaner I think.

Reply:
Fixed